lib/everypolitician/popolo/entity.rb

```diff
@@ -26,6 +26,7 @@ def key?(key)
       end
 
       def ==(other)
+        return false unless other.instance_of?(self.class) 
```

Comment:
@ondenman is there a reason why you used this approach rather than, say, `self.class == other.class`? The latter would feel more natural to me, and seems like it would be a little easier to read, especially if rolled in with the next check too (i.e. `id == other.id && self.class = other.class`), so curious whether that's something you explicitly decided against?